dora/core/server/worker/src/test/java/alluxio/worker/dora/DoraMetaManagerTest.java

```diff
@@ -57,7 +57,7 @@ public void before() throws IOException {
     PagedDoraWorker worker = mock(PagedDoraWorker.class);
     CacheManager cacheManager = mock(CacheManager.class);
     mDoraUfsManager = mock(DoraUfsManager.class);
-    mManager = new DoraMetaManager(conf, worker, cacheManager, mDoraUfsManager);
+    mManager = new DoraMetaManager(conf, cacheManager, mDoraUfsManager);
```

Comment:
this line is one of the core change in this PR. The PagedDoraWorker no longer passes its `this` out in its own constructor. The `this` ref escaping in the constructure is a bad practice we should avoid.